doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -464,14 +468,25 @@ Generic Syntax for Quotations
 In general, quotations can be introduced in terms using the following syntax, where
 :production:`quotentry` is some parsing entry.
 
-.. prodn::
-   ltac2_term += @ident : ( @quotentry )
 
 .. _ltac2_built-in-quotations:
 
 Built-in quotations
 +++++++++++++++++++
 
+.. insertprodn ltac2_quotations ltac1_expr_in_env
+
+.. prodn::
+   ltac2_quotations ::= ident : ( @lident )
+   | constr : ( @term )
+   | open_constr : ( @term )
+   | pattern : ( @cpattern )
+   | reference : ( {| & @ident | @qualid } )
+   | ltac1 : ( @ltac1_expr_in_env )
+   | ltac1val : ( @ltac1_expr_in_env )
+   ltac1_expr_in_env ::= @ltac_expr
+   | {* @ident } |- @ltac_expr
+
 The current implementation recognizes the following built-in quotations:
```

Comment:
In the list below, the `ltac1` / `ltac1val` quotations are missing. They should be added with a link to where they are documented (subsections "Simple API" and "Low-level API" of "Ltac1 from Ltac2" section).